lib/aika/accounts/user.ex

```diff
@@ -22,7 +22,7 @@ defmodule Aika.Accounts.User do
     user
     |> cast(attrs, [:email, :role, :password, :token])
     |> validate_required([:email, :password, :role])
-    |> validate_format(:email, ~r/@/)
+    |> validate_format(:email, ~r/^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4}$/)
```

Comment:
Can we keep this as it is, writing an accurate email regex is almost impossible and it's just better to check for @ sign instead. This article has good reasons as to why: https://davidcel.is/posts/stop-validating-email-addresses-with-regex/